openedx/features/enterprise_support/utils.py

```diff
@@ -183,8 +183,10 @@ def update_third_party_auth_context_for_enterprise(request, context, enterprise_
             line_break=HTML('<br/>')
         )
 
+    # The hinted login frontend `HintedLoginView.js`, expects a list of providers
+    # so not clearing `providers` here anymore. It is unclear if removing the `secondaryProviders`
+    # is necessary either.
     if enterprise_customer:
-        context['data']['third_party_auth']['providers'] = []
```

Comment:
removing this does make me slightly wary, in that I do not know where else this context is used (other than rendering that template), if at all. If there is any context I am missing lmk (also I don't know if that secondaryProviders line needs to stay either)

Reply:
I found this PR where this change was added, going to have to confirm nothing else about auth is broken by doing this https://github.com/edx/edx-platform/pull/15025